demisto_sdk/main.py

```diff
@@ -272,11 +287,115 @@ def format(file_type, **kwargs):
+@click.option(
+    "--debug-path", help="The path to save the debug file at, if not specified the debug file will be printed to the "
+                         "terminal")
+def run(**kwargs):
+    runner = Runner(**kwargs)
+    return runner.run()
+
+
 @main.resultcallback()
 def exit_from_program(result=0, **kwargs):
     sys.exit(result)
 
 
+# ====================== run-playbook ====================== #
+@main.command(name="run-playbook",
+              short_help="Run a playbook in Demisto. "
+                         "DEMISTO_API_KEY environment variable should contain a valid Demisto API Key. "
+                         "Example: DEMISTO_API_KEY=<API KEY> demisto-sdk run-playbook -p 'p_name' -u "
+                         "'https://demisto.local'.")
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    '--url', '-u',
+    help='URL to a Demisto instance. You can also specify the URL as an environment variable named: DEMISTO_BASE_URL'
+)
+@click.option(
+    '--playbook_id', '-p',
+    help="The playbook ID to run.",
+    required=True
+)
+@click.option(
+    '--wait', '-w',
+    type=click.Choice(["True", "False"]),
+    default="True",
+    show_default=True,
+    help="Wait until the playbook run is finished and get a response."
+)
+@click.option(
+    '--timeout', '-t',
+    default=90,
+    show_default=True,
+    help="Timeout for the command. The playbook will continue to run in Demisto"
+)
+def run_playbook(**kwargs):
+    playbook_runner = PlaybookRunner(**kwargs)
+    return playbook_runner.run_playbook()
+
+
+@main.command(name="json-to-outputs",
+              short_help='''Demisto integrations/scripts have a YAML file that defines them.
+Creating the YAML file is a tedious and error-prone task of manually copying outputs from the API result to the 
+file/UI/PyCharm. This script auto generates the YAML for a command from the JSON result of the relevant API call.''')
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    "-c", "--command", help="Command name (e.g. xdr-get-incidents)", required=True)
+@click.option(
+    "-i", "--infile", help="Valid JSON file path. If not specified then script will wait for user input in the "
+                           "terminal", required=False)
+@click.option(
+    "-p", "--prefix", help="Output prefix like Jira.Ticket, VirusTotal.IP, the base path for the outputs that the "
+                           "script generates", required=True)
+@click.option(
+    "-o", "--outfile", help="Output file path, if not specified then will print to stdout", required=False)
+@click.option(
+    "-v", "--verbose", is_flag=True, help="Verbose output - mainly for debugging purposes")
+@click.option(
+    "--interactive", help="If passed, then for each output field will ask user interactively to enter the "
+                          "description. By default is interactive mode is disabled", is_flag=True)
+def json_to_outputs_command(**kwargs):
+    json_to_outputs(**kwargs)
```

Comment:
```suggestion
    return json_to_outputs(**kwargs)
```

Reply:
the function don't return anything? why `return`?